api/endpoints/v2/tasks.py

```
@@ -0,0 +1,578 @@
+                detail="Cannot create task hierarchy - would create cycle or invalid relationship"
+            )
+        db.commit()
+        db.refresh(db_task)
+
+    except HTTPException:
+        raise
+    except Exception as e:
+        db.rollback()
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail=f"Failed to create subtask: {str(e)}"
```

Ruff (0.15.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)